mscore/file.cpp

```diff
@@ -1602,6 +1602,7 @@ void MuseScore::exportFile()
 #endif
       fl.append(tr("Standard MIDI File") + " (*.mid)");
       fl.append(tr("MusicXML File") + " (*.xml)");
+      fl.append(tr("MusicXML 3.1+ File") + " (*.musicxml)");
```

Comment:
Wouldn't that mean that MuseScore's MusicXML couldn't be opened by older versions of other software?